src/default/assets/js/src/typedoc/services/Viewport.ts

```diff
@@ -61,6 +85,21 @@ namespace typedoc
         onScroll() {
             this.scrollTop = $window.scrollTop() || 0;
             this.trigger('scroll', this.scrollTop);
+            this.hideShowToolbar();
+        }
+
+
+        /**
+         * Handle hiding/showing of the toolbar.
+         */
+        hideShowToolbar() {
+            const isShown = this.showToolbar;
+            this.showToolbar = this.lastY >= this.scrollTop || this.scrollTop === 0;
+            if (isShown !== this.showToolbar) {
+                this.toolbar.classList[this.showToolbar ? 'remove' : 'add']('tsd-page-toolbar--hide');
```

Comment:
Is there a reason to not use `classList.toggle` here?

Reply:
I did it that way because I wanted the displaced state to not be the default on the nav. I've switched that around in the CSS so we can use toggle on both of these. Other option was putting it as a class in the markup file.